assign-2/src/data/CreateStudentMenuItem.java

```diff
@@ -0,0 +1,32 @@
+package data;
+
+import data.MenuItem;
+import data.StudentDao;
+import logic.Student;
+
+import java.util.Scanner;
+
+public class CreateStudentMenuItem extends MenuItem {
```

Comment:
This class should not be inside the `data` layer. Move it to the `presentation` layer instead.

Reply:
I will move it to the presentation layer, Thank you!